.gitignore

```diff
@@ -10,3 +10,5 @@ delstats
 *.pyc
 *.created
 tmp-cni
+k8s-install/scripts/tmp/
```

Comment:
I was imagining using something like `mktemp`: https://www.mktemp.org/manual.html

That way we don't need to check in a `tmp` directory in the repo. WDYT?

Reply:
Well this is explicitly ignoring it so the directory is not checked in. But are you thinking we run `mktmp` in the make target and had it off to the script?